style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -227,6 +227,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorBackgroundDialog: {
+    description: 'The background color of the dialog.',
```

Comment:
which dialog windows is this getting applied to? modals?

Reply:
I see. Maybe at least have more guidance in the description? Otherwise, it would be difficult to navigate design tokens table without this context